dtfs-central-api/api-tests/helpers/set-object-property-value-from-string-path.js

```diff
@@ -0,0 +1,14 @@
+const setObjectPropertyValueFromStringPath = (object, path, value) => {
```

Comment:
Where is this used? This feels a little off to me that we need to do this so just want to review if we can avoid having this kind of function!

Reply:
This is used in `dtfs-central-api\api-tests\v1\tfm\deal\tfm-deals-get-sorting.api-test.js` to set the properties of test deals that will be inserted into the db using the same string path that will then be used in the query parameters to sort these deals. It's to keep the tests DRY and easily extendable.